docs/evaluation/how_to_guides/index.md

```diff
@@ -1,63 +1,85 @@
 # Evaluation how-to guides
 
-Step-by-step guides that cover key tasks and operations for doing evaluating and testing your applications in LangSmith.
+These guides answer “How do I….?” format questions. 
+They are goal-oriented and concrete, and are meant to help you complete a specific task. 
+For conceptual explanations see the [Conceptual guide](./concepts). 
+For end-to-end walkthroughs see [Tutorials](./tutorials). 
+For comprehensive descriptions of every class and function see the [API Reference](https://langsmith-sdk.readthedocs.io/en/latest/evaluation.html).
+
+## Offline evaluation
+
+Evaluate and improve your application.
+
+### Run an evaluation
+- [Run an evaluation using the SDK](./how_to_guides/evaluation/evaluate_llm_application)
+- [Run an evaluation asynchronously](./how_to_guides/evaluation/async)
+- [Run an evaluation comparing two experiments](./how_to_guides/evaluation/evaluate_pairwise)
+- [Run an evaluation of a LangChain / LangGraph object](./how_to_guides/evaluation/langchain_runnable)
```

Comment:
Probably worth it to cover LangChain and LangGraph in separate guides?

There's some more interesting things in LG like evaluating a single node, traectory, etc. It might be worth it to call out the documentation for running a larger eval in the LG eval guide.

We have this tutorial I hacked together with Lance a while back but it's definitely overly complicated and I wonder if bits of it can be adapted to a more concise how to guide https://docs.smith.langchain.com/evaluation/tutorials/agents

Reply:
i kinda think this should live in langgraph docs and be linked from here